src/components/TextField/TextField.module.scss

```diff
@@ -326,3 +326,16 @@
 .textField .inputWrapper.wrapperSizeLarge .iconContainer .icon {
   font-family: var(--font-family);
 }
+
+input[type="date"]::-webkit-calendar-picker-indicator,
+input[type="datetime-local"]::-webkit-calendar-picker-indicator {
+  display: block;
+  background: url("./assets/Calendar.svg") no-repeat;
+}
+
+:global(.dark-app-theme, .black-app-theme, .hacker_theme-app-theme) {
+  input[type="date"]::-webkit-calendar-picker-indicator,
+  input[type="datetime-local"]::-webkit-calendar-picker-indicator {
+    background: url("./assets/CalendarInverted.svg") no-repeat;
+  }
+}
```

Comment:
the display block on only one of them is intentional?

Reply:
Removed it completely